src/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties

```diff
@@ -39,17 +39,29 @@ ascanbeta.HTTPParamPoll.extrainfo = http://www.google.com/search?q=http+paramete
 ascanbeta.insecurehttpmethod.trace.exploitable.desc = The insecure HTTP method [{0}] is enabled for this resource, and is exploitable. The TRACK and TRACE methods may be used by an attacker, to gain access to the authorisation token/session cookie of an application user, even if the session cookie is protected using the 'HttpOnly' flag. For the attack to be successful, the application user must typically be using an older web browser, or a web browser which has a Same Origin Policy (SOP) bypass vulnerability.
+ascanbeta.insecurehttpmethod.trace.exploitable.extrainfo = A TRACE request was sent for this request, with a custom cookie value [{0}]. This cookie value was disclosed in the HTTP response, confirming the vulnerability.
 ascanbeta.insecurehttpmethod.connect.exploitable.desc = The insecure HTTP method [{0}] is enabled for this resource, and is exploitable. It was found to be possible to establish a tunneled socket connection to a third party service, using this HTTP method.  This would allow the service to be used as an anonymous spam relay, or as a web proxy, bypassing network restrictions. It also allows it to be used to establish a tunneled VPN, effectively extending the network perimeter to include untrusted components. 
 ascanbeta.insecurehttpmethod.extrainfo = The OPTIONS method disclosed the following enabled HTTP methods for this resource: [{0}]
-ascanbeta.insecurehttpmethod.trace.exploitable.extrainfo = A TRACE request was sent for this request, with a custom cookie value [{0}]. This cookie value was disclosed in the HTTP response, confirming the vulnerability.
 ascanbeta.insecurehttpmethod.connect.exploitable.extrainfo = The CONNECT method was used to establish a socket connection to [{0}], via the web server.
 ascanbeta.insecurehttpmethod.soln = Disable insecure methods such as TRACK, TRACE, and CONNECT on the web server, and ensure that the underlying service implementation does not support insecure methods.
+ascanbeta.insecurehttpmethod.delete.exploitable.desc = This method was originally intended for file managemant operations. It is now most commonly used in REST services, It is used to **delete** a resource identified by a URI, On successful deletion, return HTTP status 200 (OK) along with a response body.
```

Comment:
> , It is used to **delete** a resource identified by a URI, On successful deletion, return HTTP status 200 (OK) along with a response body.

I don't think we need to explain the semantics of the methods in detail, just mention that they are enabled and that the user should confirm that's expected. @kingthorin wdyt?

Reply:
Agreed